lib/utils.py

```diff
@@ -485,6 +500,145 @@ def get_client_dn(proxy: Union[None, str] = None) -> Union[str, Any]:
     return ""
 
 
+# # py li nt: disable=chained-comparison,too-many-return-statements,too-many-branches,too-many-statements
+def check_space(
+    path: str, min_kblocks: int = 100, min_files: int = 20, verbose: int = 0
+) -> bool:
+    """check if there is room / quota in location"""
+    fs: str = ""
+    d_ok: bool = False
+
+    fs, d_ok = check_space_df(path, min_kblocks, min_files, verbose)
+
+    if verbose > 1:
+        sys.stderr.write(f"df check: {d_ok}, fs: '{fs}'\n")
+
+    if fs and d_ok:
+        q_ok = check_space_quota(path, fs, min_kblocks, min_files, verbose)
+    else:
+        q_ok = True
+
+    return d_ok and q_ok
+
+
+def check_space_df(
+    path: str, min_kblocks: int = 100, min_files: int = 20, verbose: int = 0
+) -> Tuple[str, bool]:
+    """check for enough disk space at path, also return fs mountpoint"""
+    cmd = f"df -k {path}"
+    # sample out:
+    # ====================
+    # Filesystem                      1K-blocks        Used Available Use% Mounted on
+    # if-nas-0.fnal.gov:/lbnewc/app 21474836480 20932591360 542245120  98% /dune/app
```

Comment:
I hate to ask this, but my understanding of the current DOE policy is that we should try to avoid having hostnames out on the public web.  Could you please swap in <HOSTNAME> or something like that in there?